libraries/AP_HAL_ChibiOS/hwdef/airbotf4/hwdef.dat

```diff
@@ -0,0 +1,128 @@
+# hw definition file for processing by chibios_pins.py
+# Omnibus F4 PRO, Omnibus F4 V3 boards
+# with F405 mcu, mpu6000 imu, bmp280 barometer, 7456 series osd  and sdcard
+
+MCU STM32F4xx STM32F405xx
+
+HAL_CHIBIOS_ARCH_F405 1
+
+# board ID for firmware load
+APJ_BOARD_ID 70
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+STM32_PLLM_VALUE 8
+
+# board voltage
+STM32_VDD 330U
+
+STM32_ST_USE_TIMER 5
+
+# flash size
+FLASH_SIZE_KB 1024
+FLASH_RESERVE_START_KB 64
+
+CCM_RAM_SIZE_KB 64
+RAM_SIZE_KB 128
+
+# order of I2C buses
+I2C_ORDER I2C2
+
+# order of UARTs
+UART_ORDER OTG1 USART6 USART1
+
+#adc
+PC1 BAT_CURR_SENS ADC1 SCALE(1)
+PC2 BAT_VOLT_SENS ADC1 SCALE(1)
+PA0 RSSI_IN ADC1
+
+#pwm output
+PB0 TIM1_CH2N TIM1 PWM(1) GPIO(26)
+PB1 TIM1_CH3N TIM1 PWM(2) GPIO(27)
+PA3 TIM2_CH4 TIM2 PWM(3) GPIO(17)
+PA2 TIM2_CH3 TIM2 PWM(4) GPIO(16)
+PA1 TIM2_CH2 TIM2 PWM(5) GPIO(15)
+PA8 TIM1_CH1 TIM1 PWM(6) GPIO(41)
+
+PA4 MPU6000_CS CS
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PA7 SPI1_MOSI SPI1
+
+PB10 I2C2_SCL I2C2
+PB11 I2C2_SDA I2C2
+
+#PB15 SPI2_MOSI SPI2
+#PB14 SPI2_MISO SPI2
+#PB13 SPI2_SCK SPI2
+#PB12 SDCARD_CS CS
+
+PA10 USART1_RX USART1
+PA9 USART1_TX USART1
+
+PC6 USART6_TX USART6
+PC7 USART6_RX USART6
+
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+#PA15 OSD_CS CS
+#PB3 BMP280_CS CS
```

Comment:
SPI3 is used to connect winbond spi flash.
It currently not supported, but I think it should be enableb.
Something like
PB3 FLASH_CS CS
....